src/App.tsx

```diff
@@ -1,155 +1,105 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+
+import classNames from 'classnames';
+
+import { getTodos } from './api/todos';
+
+import { SortType } from './types/SortType';
+import { Todo } from './types/Todo';
+
 import { UserWarning } from './UserWarning';
+import { filterTodos } from './utils/helpers';
 
-const USER_ID = 0;
+import { TodoList } from './components/TodoList';
+import { AddingTodo } from './components/AddingTodo';
+import { TodoFooter } from './components/TodoFooter';
+
+const USER_ID = 6749;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [newTitle, setNewTitle] = useState('');
+  const [sortType, setSortType] = useState<SortType>(SortType.ALL);
+  const [error, setError] = useState('');
+
+  useEffect(() => {
+    const fetchTodos = async () => {
+      try {
+        const fetchedTodos = await getTodos(USER_ID);
+
+        setTodos(fetchedTodos);
+      } catch {
+        setError('fetch');
+
+        setTimeout(() => {
+          setError('');
+        }, 3000);
+      }
+    };
+
+    fetchTodos();
+  }, []);
+
   if (!USER_ID) {
     return <UserWarning />;
   }
 
+  const handleCloseError = () => {
+    setError('');
+  };
+
+  const handleTitle = (title: string) => {
```

Comment:
```suggestion
  const handleChangeTitle = (title: string) => {
```
or some another variant. cause don't understand what that handler doing